src/network/index.ts

```diff
@@ -197,6 +188,8 @@ export class NetworkClass extends EventEmitter {
         hashKey: this.shardusCryptoHashKey,
         signingSecretKeyHex: this.signingSecretKeyHex,
       },
+      payloadSizeLimitInBytes: config.p2p.payloadSizeLimitInBytes,
```

Comment:
might be nice to have this under a config object given that other variables with similarity in use case are also under an object structure for the same function definition .

Reply:
Yes, it makes sense. It is updated, thanks!